app/views/schedules/edit.html.erb

```diff
@@ -5,8 +5,9 @@
   <div class="col-md-8 col-md-offset-2">
     <%= form_for(@schedule) do |f| %>
         <%= render partial: 'fields', locals: {f: f, object: @schedule} %>
-        <%= button_tag 'Save', type: :submit, class: 'btn btn-sm btn-primary' %>
-        <%= link_to 'Cancel', schedules_path, class: 'btn btn-sm btn-primary btn-danger' %>
+
+        <%= button_to 'Save', edit_schedule_path(@schedule), class: 'btn btn-sm btn-primary' %>
```

Comment:
Correct, that is the change you've made. What I'm saying is that it is working as intended, so there is no need for that change at the moment. You'll notice that redirecting to the index page after saving on Edit is the standard workflow for most (all?) controllers in the project. Saving and staying on the Edit page is a new use case and if we're changing how we do things we should probably at least write up an activity diagram or some such and brainstorm the right way to go about it.

Case in point, if you'd really like a way to save and stay on the same page, maybe a new button with something like "Save and Continue Editing" would do the trick rather than changing the current button. Or, if we want to get really fancy, you could use Bootstrap to convert the current Save button to a button with a dropdown (see "Add Equipment" on Equipment Profiles) and have it do the default action when you click and offer the "And Continue" option drop down when you click the carat.